src/styles/pages/index.scss

```diff
@@ -139,4 +139,7 @@
       }
     }
   }
+  .swiper {
```

Comment:
Penso si possa evitare, o mettendo un className a `<Swiper>` (se lo supporta) o wrappandolo in una div con `pb-3`